sno/working_copy/postgis.py

```diff
@@ -299,7 +331,7 @@ def create(self):
                 )
             )
 
-    def delete(self):
+    def delete(self, keep_container_if_possible=False):
         """ Delete all tables in the schema"""
         # We don't use drop ... cascade since that could also delete things outside the schema.
```

Comment:
Because tables can depend on each other with foreign keys, and we don't drop them in any particular order, it's quite likely this function can fail because of FK constraints, even though all the tables will eventually be dropped.

Probably this could be avoided with `SET CONSTRAINTS ALL DEFERRED`

Reply:
Done